### PR TITLE
Integration with backend and already verified modal added

### PR DESCRIPTION
**Pull request by NicoleTobal**
- Kyc dialog integrated with backend: files are uploaded and then de files ids and the random code are saved while changer user kyc status to pending
- An “already verified” dialog is shown if user has already completed kyc
- Kyc dialog is also opened when clicking on the  level up button from accounts
- A random code of four alphanumeric uppercase characters is generated on the Upload Dialog

### CODE REVIEW

File: src/components/profile/kyc/UploadDialog.jsx [hunk 1]
```diff
@@ -105,14 +134,14 @@ export default function UploadDialog(props) {
...
-                                image={imgselfie}
+                                image={imgselfie.image}
                                 title="Selfie"
                             >
                             </CardMedia>
                             <CardContent className={classes.cardContent}>
                                 Selfie del titular con su documento y
                                 una nota escrita con la frase
-                                "Usuario Veinte.net XXX"
+                                "Usuario Veinte.net {randomCode}"
```
Comment (ramyx) [inline src/components/profile/kyc/UploadDialog.jsx]: El ramdoncode deberia generarse por backend si se quiere asegurar la no repeticion.

File: src/api/backend.js [hunk 2]
```diff
@@ -227,3 +227,54 @@ export const withdraw = async (amount, coinId, paymentMethodId, extraField) => {
     return new Maybe({ err });
   }
 };
+
```
Comment (ramyx) [inline src/api/backend.js]: En lugar de backend se deberia usar un archivo "services" para cada recurso del backend

Comment (NicoleTobal) [reply src/api/backend.js]: https://trello.com/c/PbHmWPXJ/168-agregar-services-para-cada-llamada-a-un-endpoint-de-backend

File: src/components/profile/kyc/UploadDialog.jsx (hunk 1 above)
Comment (NicoleTobal) [reply src/components/profile/kyc/UploadDialog.jsx]: https://trello.com/c/PbHmWPXJ/168-agregar-services-para-cada-llamada-a-un-endpoint-de-backend

**Pull request merged**